### PR TITLE
adding stargazer and xtable examples

### PR DESCRIPTION
**Pull request by mhines-usgs**
also removing visit status

### CODE REVIEW

File: R/sitevisitpeak-data.R [hunk 1]
```diff
@@ -29,8 +28,9 @@ sitevisitpeakTable <- function(rawData){
                    "Verification Comments",
                    "Corrected Value",
                    "Qualifier",
-                   "Date",
-                   "Time",
+                   "Date 
+                   ",
```
Comment (lindsayplatt) [inline R/sitevisitpeak-data.R]: is this meant to be a separate line or typo?

Comment (mhines-usgs) [inline R/sitevisitpeak-data.R]: eh weird, fix coming up.

Comment (lindsayplatt) [inline R/sitevisitpeak-data.R]: :+1:

File: inst/sitevisitpeak/sitevisitpeak.Rmd [hunk 2]
```diff
@@ -1,14 +1,16 @@
...
 geometry: top=.5cm, bottom=.5cm, left=1cm, right=1cm
 ---
 
 ![](http://www.usgs.gov/newsroom/images/2015-04-07/usgs_logo.png)
 
-## `r getReportMetadata(ts,'title',required=TRUE)` Report
+**`r getReportMetadata(ts,'title',required=TRUE)` Report**
 
-**Location:**  `r getReportMetadata(data,'stationId',required=TRUE)` `r getReportMetadata(data,'stationName',required=TRUE)`
+**Location:**  `r getReportMetadata(data,'locationNumber',required=TRUE)` `r getReportMetadata(data,'locationName',required=TRUE)`
```
Comment (mhines-usgs) [inline inst/sitevisitpeak/sitevisitpeak.Rmd:11]: old way for now. @thongsav-usgs

File: inst/sitevisitpeak/sitevisitpeak.Rmd [hunk 3]
```diff
@@ -1,14 +1,16 @@
...
+
+**Sub-Location:** `r getReportMetadata(data,'sublocation',required=TRUE)`
```
Comment (mhines-usgs) [inline inst/sitevisitpeak/sitevisitpeak.Rmd:13]: from the mockup and the jira ticket I see this in the header @thongsav-usgs 

https://internal.cida.usgs.gov/jira/browse/AQCU-265

**Pull request closed without merging**